easybuild/easyconfigs/p/PyMOL/PyMOL-2.5.0-foss-2020b.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'PythonPackage'
```

Comment:
I suggested creating a `PythonBundle` instead. Have a look at e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/matplotlib/matplotlib-3.5.2-foss-2022a.eb

We mostly create PythonBundle's now, it avoids having to set `exts_defaultclass` and you can set `use_pip` and `sanity_pip_check` as global keywords (wihtout needing to nest it in a `exts_default_options`  dict). Also, it means you only need to set those once, instead of at two levels (as you do now).

All in all, the `PythonBundle` EasyBlock makes the EasyConfig look a bit cleaner.

Reply:
Agreed, do let us know if you need any help with that @guacke!